scripts/build/Platform/Linux/build_config.json

```diff
@@ -387,12 +387,39 @@
       "CMAKE_TARGET": "install"
     }
   },
+  "install_20_release": {
```

Comment:
You may ant to rename thus `install_20_mono_release` to document that this builds the monolithic permutation of release